ChopChop/Models/Ingredients/Quantity.swift

```diff
@@ -0,0 +1,258 @@
+/**
```

Comment:
This file is inevitably quite duplicated but I think we somehow need to explain the duplicated code in the DG

Reply:
Yes I agree, I'll see if I can reduce the duplication by using `QuantityType`.